src/controllers/sandbox.ts

```diff
@@ -93,8 +93,11 @@ const sandboxCache = cacheSandboxDescriptor(async (taskID) => {
 export default function (app: Express.Application) {
     app.get('/api/sandbox/*', async function (req: Request, res: Express.Response, next: Express.NextFunction) {
         try {
+            if (req.user == undefined)
```

Comment:
I thought we had authenticated access to the sandbox (without authorization).

Reply:
Hmm yes you're right! Maybe I can get the information from another source. I'm going to check this

Reply:
Yup I checked with `AUTHORIZE_ALL_SANDBOXES=true` and I see user names, so the patch is correct